HLTrigger/Configuration/python/HLT_75e33/eventsetup/hltESPOnlineBeamSpotProducer_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+hltOnlineBeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer",
+                                             timeThreshold = cms.int32( 48 ),
+                                             sigmaZThreshold = cms.double( 2.0 ),
+                                             sigmaXYThreshold = cms.double( 4.0 )
```

Comment:
Just being pedantic, all other files use a 4-space indentation:
```suggestion
    timeThreshold = cms.int32( 48 ),
    sigmaZThreshold = cms.double( 2.0 ),
    sigmaXYThreshold = cms.double( 4.0 )
```